docs/config/ping-federate-example.md

```diff
@@ -35,121 +41,148 @@ For example:
+        1. Choose **POST** from the dropdown menu under **BINDING**.
+        2. In **ENDPOINT URL**, add the value of **Redirect URI**.
+        3. Click add. 
+  2. On the **SLO Service URLs** tab:
+        1. Choose **POST** from the dropdown menu under **BINDING**.
+        2. In **ENDPOINT URL**, add the value of **Redirect URI**.
+        3. In **RESPONSE URL**, add the value of **Redirect URI**.
+        4. Click **add**.
+  3. On the **Allowable SAML Bindings** tab untick all of the checkboxes except for **POST**.
+  4. On the **Signature Policy** tab tick **Require AuthN requests to be signed when received via the POST or Redirect bindings**.
+  5. On the **Encryption Policy** tab ensure that the **None** checkbox is ticked.
+  6. On the **Summary** tab verify that the values are as per the following screenshot.
+  ![Protocol Settings Summary](./slo-default-url.png)
+  7. Click **Done** to return to the browser SSO configuration tabs.
+
+  8. On the **Summary** tab click **Done** to return to the main SP connection configuration tabs.
+
+  9. On the **Credentials** tab click **Configure Credentials** which will launch a new set of tabs for configuring the credentials. 
+
+ #### Configuring credentials
+ 1. On the **Digital Signature Settings** tab, select your organization's key pair and certificate from the **Signing Certificate** dropdown. 
+ 2. On the **Signature Verification Settings** tab, click **Manage Signature Verification Settings** which will launch a new set of tabs for the signature verification settings.
+
+  #### Signature verification settings
+  1. On the **Trust Model** tab select *Unanchored*.
+  2. On the **Signature Verification Certificate** tab click **Manage Certificate**.
+  3. Click **Import** and browse for the `.cert` file you created earlier in the **Filename** field.
+  4. Note the imported certificate's **Serial Number** similar to the following screenshot.
+  ![Import Certificate Summary](./import-cert-summary.png)
+  5. Click **Done**.
+  6. Still on the **Signature Verification Certificate** tab select the certificate with the ID (serial number) from the previous step from the **Primary** dropdown.
+  7. Click **Done** to return to the credentials configuration tabs.
+
+
+**Export configuration settings**
+1. On the **Summary** tab click **Done** to return to the main SP connection configuration tabs.
+
+2. On the **Activation & Summary** tab tick the **Active** checkbox in the **Connection Status** row and **Save**. 
+
+3. Back on the identity service homepage, click **Manage all SP**.
+4. Your new connection should now appear in the **SP Connections** list. To the right of it click **Export metadata**.
+5. In the summary tab, click **Export**.
+
+
+### Identity Service Configuration
+Back in the Identity Service tab that you left open:
+1. Chose the file that you downloaded as part of the **Export metadata** step.
+
+    Now you should see that all your PingFederate configurations have been populated.
```

Comment:
Does this need to be separated from the previous sentence with a blank line? IMHO it doesn't, it's a logical continuation of the previous sentence.